app/main.py

```diff
@@ -1,25 +1,87 @@
 from abc import ABC
+from typing import Any, Type
+
+
+class SlideLimitationValidator(ABC):
+    def __init__(self,
+                 age: int,
+                 height: int,
+                 weight: int,
+                 ) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class IntegerRange:
-    pass
+    def __init__(self,
+                 min_amount: int,
+                 max_amount: int,
+                 ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
 
+    def __set_name__(self, owner: Type[Any], name: str) -> None:
+        self.protected_name = "_" + name
 
-class Visitor:
-   pass
+    def __get__(self,
+                instance: SlideLimitationValidator,
+                owner: Type[Any]
+                ) -> int:
+        return getattr(instance, self.protected_name)
 
+    def __set__(self,
+                instance: SlideLimitationValidator,
+                value: int,
+                ) -> None:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(f"Value should be less"
+                             f" than {self.max_amount} "
+                             f"and greater than {self.min_amount}")
+        setattr(instance, self.protected_name, value)
 
-class SlideLimitationValidator(ABC):
-    pass
+
+class Visitor:
+    def __init__(self,
+                 name: str,
+                 age: int,
+                 weight: int,
+                 height: int
+                 ) -> None:
+        self.name = name
+        self.age = age
+        self.height = height
+        self.weight = weight
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    height = IntegerRange(80, 120)
+    weight = IntegerRange(20, 50)
 
 
 class AdultSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(14, 60)
+    height = IntegerRange(120, 220)
+    weight = IntegerRange(50, 120)
 
 
 class Slide:
-    pass
+    def __init__(self,
+                 name: str,
+                 limitation_class: SlideLimitationValidator,
```

Comment:
Why you change this?????
Undo it. It accepts type 

The error was here
<img width="872" alt="image" src="https://github.com/mate-academy/py-aquapark/assets/65166203/a49b54fa-efe8-4df5-ba2a-f0d5b0055389">